uks/uks/settings.py

```diff
@@ -39,11 +39,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'apps',
```

Comment:
We don't have module apps anymore, so this is redundant

Reply:
'apps' is now removed.